aztarna/cmd.py

```diff
@@ -47,6 +48,8 @@ def main():
                 scanner.initialize_shodan()
         elif args.type.upper() == 'ROSIN':
             scanner = ROSIndustrialScanner()
+        elif args.type == 'ROS2' or args.type == 'ros2':
```

Comment:
We need to have a look on this:

```bash
root@676deddf42e8:~/aztarna# aztarna -t ros2
aztarna.cmd - Exception occurred during execution
Traceback (most recent call last):
  File "/usr/local/bin/aztarna", line 11, in <module>
    load_entry_point('aztarna==1.2', 'console_scripts', 'aztarna')()
  File "/usr/local/lib/python3.6/dist-packages/aztarna-1.2-py3.6.egg/aztarna/cmd.py", line 91, in main
    if stdout is not None:
  File "/usr/local/lib/python3.6/dist-packages/aztarna-1.2-py3.6.egg/aztarna/cmd.py", line 65, in main
    identchars = IDENTCHARS
  File "/usr/local/lib/python3.6/dist-packages/aztarna-1.2-py3.6.egg/aztarna/ros/ros2/scanner.py", line 19, in scan_pipe_main
NotImplementedError
```

Reply:
Fixed via: https://github.com/aliasrobotics/aztarna/pull/26/commits/2c8a9749fe40af826dba5a7e87789a0dfe4b27ae